events/tests/test_event_get.py

```diff
@@ -1331,3 +1338,26 @@ def test_sort_events_by_minimum_attendee_capacity(api_client, event, event2, eve
     assert results[0]["id"] == event3.id
     assert results[1]["id"] == event2.id
     assert results[2]["id"] == event.id
+
+
+@pytest.mark.parametrize("hide_children", [True, False])
+@pytest.mark.django_db
+def test_get_event_list_hide_recurring_children(api_client, hide_children):
+    now = timezone.now()
+    event_1 = EventFactory(origin_id=1, start_time=now)
+    event_2 = EventFactory(data_source=event_1.data_source, origin_id=2, start_time=now)
```

Comment:
Do the `origin_id` and `start_time` values matter in this test? Can you leave them blank?